### PR TITLE
All CSS Slide in Form labels on Address Fields.

### PR DESCRIPTION
**Pull request by MatthewKennedy**
Added form labels that appear once text is entered into the form fields giving users some feedback as to what the field was asking for, before text was entered and the placeholder disappears.

Blank
Screenshot 2020-06-03 at 17 12 33

Clicked or Tabbed into field
Screenshot 2020-06-03 at 17 12 54

Filled in
Screenshot 2020-06-03 at 17 13 44

### CODE REVIEW

**Comment by damianlegawiec** [comment]
Thank you @MatthewKennedy - this looks awesome!

**Comment by MatthewKennedy** [comment]
Hi @damianlegawiec there will be some Code Climate fixes still to do, and as always, I left a `console.log` in the JS file.

Do you want me to put the slide in labels on more form fields such as login fields or just leave it on the address form for now?

**Comment by damianlegawiec** [comment]
@MatthewKennedy let's start with address form and move from that to all of the others in sub-sequent PRs, thank you!

**Comment by MatthewKennedy** [comment]
@damianlegawiec No problem, I'll just get this PR in good shape to start with and keep it just on the address form.

**Comment by MatthewKennedy** [comment]
## To Do
- Fix zip code required indicators

**Review by AleksandraKaminska — APPROVED** [review]

**Review by kaliszKrzysztof — APPROVED** [review]

**Comment by MatthewKennedy** [comment]
### Since Last Checked:
- Fix Zip Code required/not required indication in placeholder and label (all local translation safe).
- Fix select element height, (was slightly smaller than input field causing a jump when swapping field type)
- Use translatable local for required notation at bottom of form, not a * in the html
- Fix some small styling issues, all input and selector elements use the the same black outline for focus.
- Tidy code up further.

File: frontend/app/helpers/spree/addresses_helper.rb [hunk 1]
```diff
@@ -13,23 +13,49 @@ def address_field(form, method, address_id = 'b', &handler)
...
+                          aria: { label: method_name }) +
+            form.label(method_name,
+                       is_required ? "#{method_name} #{required}" : method_name,
+                       class: 'text-uppercase')
         end
       end
     end
 
+    def address_zipcode(form, country, _address_id = 'b')
+      country ||= Spree::Country.find(Spree::Config[:default_country_id])
```
Comment (damianlegawiec) [inline frontend/app/helpers/spree/addresses_helper.rb:25]:
```suggestion
      country ||= Spree::Country.default
```

**Review by damianlegawiec — APPROVED** [review]
Looks and works great, thank you @MatthewKennedy 👍

**Pull request merged**